gr-trellis/python/trellis/bindings/viterbi_combined_python.cc

```diff
@@ -27,4 +27,47 @@ namespace py = pybind11;
 // pydoc.h is automatically generated in the build directory
 #include <viterbi_combined_pydoc.h>
 
-void bind_viterbi_combined(py::module& m) {}
+template <typename IN_T, typename OUT_T>
```

Comment:
Very nice, using templates in binding code to avoid  code duplication! Thank you.

I do have a wish: we now have new Python functionality (yay!) that we don't test (nooooo). So if you could throw in a Python qa_something file that at least instantiates every block, that would be good – that would at least make regressions that break the constructor noticeable :)

Reply:
I have no idea how to do that, but I will try :) 

Yes, I thought tests are already integrated as it was already part of gr3.8 until I saw that's not the case for the new pybind11 modules and the old tests do not exist :(

Reply:
qa_tests have been added as requested :) Not sure if "eval" is the best idea, but it saves code and shouldn't matter for qa code as it prevents code duplication.